menas/ui/components/run/runDatasetNameMaster.view.xml

```diff
@@ -0,0 +1,45 @@
+<!--
+  ~ Copyright 2018-2019 ABSA Group Limited
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<core:View id="runMasterView" xmlns:core="sap.ui.core" xmlns="sap.m"
+           controllerName="components.run.runDatasetNameMaster"
+           xmlns:cust="http://schemas.sap.com/sapui5/extension/sap.ui.core.CustomData/1">
+    <Page id="masterPage" title="Dataset Runs" showNavButton="true" navButtonPress="onPressMasterBack">
```

Comment:
I think there should be the same arrows ( `>` ) as are at the Navigation menu. For better understanding that it rolled out after pressing the button.

Reply:
![image](https://user-images.githubusercontent.com/56251097/71819287-beecd180-308b-11ea-8c76-baa1a23eb445.png)
![image](https://user-images.githubusercontent.com/56251097/71819373-03786d00-308c-11ea-9335-9c33f1ac2230.png)

Reply:
That's an excellent point! Thanks